addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommonActiveScanRuleInfo.java

```diff
@@ -28,6 +28,8 @@ public default String getHelpLink() {
     }
 
     default String getHelpLink(int id) {
+
+        Alert ID: <a href="https://www.zaproxy.org/docs/alerts/90020/">90020</a>
```

Comment:
This is the wrong place for this

Reply:
Could you let me know the correct place to address this?